tardis/visualization/plot_util.py

```diff
@@ -28,7 +28,7 @@ def axis_label_in_latex(label_text, unit, only_text=True):
     # If present, place s^{-1} just after erg
     if "erg" in unit_in_latex and "s^{-1}" in unit_in_latex:
         constituent_units = (
-            re.compile("\\\mathrm\{(.*)\}")
+            re.compile("\\\\mathrm\\{(.*)\\}")
```

Comment:
That's a lot of backslashes. I think this should be a raw string instead

Reply:
Fixed in [`6a979c8` (#2745)](https://github.com/tardis-sn/tardis/pull/2745/commits/6a979c8c32644ec077161f8e22aaaecb987a3eac)